backend/edna.cabal

```diff
@@ -115,6 +118,7 @@ executable edna-server
       Paths_edna
   hs-source-dirs:
       app
+  ghc-options: -rtsopts "-with-rtsopts=-T -I0"
```

Comment:
The `-I0` change does not seem related to this PR, so it would be better to make it separately. But, most importantly, why does it even make sense? Do we have (or even expect to have) any problems with idle GC? If we do, maybe it would be better to just increase the timeout?

Reply:
You right, it'll be better to make changes separately.

At first look for now edna is not very complicated and it seems we have no problems with idle. My motivation to add it was that it will not cause problems and this suggestion was provided from relevant area.

If there are objection - it'll be better to remove this change from current PR and discuss it in slack